add two new settings for Anomaly Detection 2.5 (#2450)
opensearch-project/documentation-website: _monitoring-plugins/ad/settings.md

plugins.anomaly_detection.dedicated_cache_size | 10 | If the real-time analysis of a high cardinality detector starts successfully, the anomaly detection plugin guarantees keeping 10 (dynamically adjustable via this setting) entities' models in memory per node. If the number of entities exceeds this limit, the plugin puts the extra entities' models in a memory space shared by all detectors. The actual number of entities varies based on the memory that you've available and the frequencies of the entities. If you'd like the plugin to guarantee keeping more entities' models in memory and if you're cluster has sufficient memory, you can increase this setting value.
plugins.anomaly_detection.max_concurrent_preview | 2 | The maximum number of concurrent previews. You can use this setting to limit resource usage.
plugins.anomaly_detection.model_max_size_percent | 0.1 | The upper bound of the memory percentage for a model.

> plugins.anomaly_detection.door_keeper_in_cache.enabled | False | When set to `true`, OpenSearch places a bloom filter in front of an inactive entity cache to filter of items that are not likely to appear more than once.

Reviewer: ```suggestion
plugins.anomaly_detection.door_keeper_in_cache.enabled | False | When set to `true`, OpenSearch places a bloom filter in front of an inactive entity cache to filter out items that are not likely to appear more than once.
```

> plugins.anomaly_detection.door_keeper_in_cache.enabled | False | We have a bloom filter placed in front of an inactive entity cache to filter out unpopular items that are not likely to appear more than once. This setting allows you to either enable the bloom filter or not.

Reviewer 2: ```suggestion
plugins.anomaly_detection.door_keeper_in_cache.enabled | False | When set to `true`,  a bloom filter activates in front of an inactive entity cache that filters out unpopular items. If the bloom filter is active, unpopular items are unlikely to appear in notifications more than once.
```

Author: It's more correct to say the bloom filter is used in front of an inactive entity cache to filter out unpopular items that are not likely to appear more than once. There aren't notification as these are just implementation details that effect how we cache things. It's also not going to lead to unpopular items to be less likely to appear. It more so filters out items from the cache that are not appearing more than once. The motivation behind this is to save memory by not storing a bunch of entities that only appear once. 

Does this sound better: "When set too `true`,  a bloom filter is placed in front of an inactive entity cache to filter out items that are not likely to appear more than once."

> plugins.anomaly_detection.door_keeper_in_cache.enabled | False | We have a bloom filter placed in front of an inactive entity cache to filter out unpopular items that are not likely to appear more than once. This setting allows you to either enable the bloom filter or not.

Reviewer: Can we say "The anomaly detection plugin has a bloom filter"? To avoid the use of "we".
```suggestion
plugins.anomaly_detection.door_keeper_in_cache.enabled | False | The anomaly detection plugin has a bloom filter placed in front of an inactive entity cache to filter out unpopular items that are not likely to appear more than once. This setting allows you to either enable or disable the bloom filter.
```

> plugins.anomaly_detection.hcad_cold_start_interpolation.enabled | False | Whether interpolation in HCAD cold start is enabled or not.

Reviewer: ```suggestion
plugins.anomaly_detection.hcad_cold_start_interpolation.enabled | False | Whether interpolation in HCAD cold start is enabled.
```

> plugins.anomaly_detection.hcad_cold_start_interpolation.enabled | False | Whether interpolation in HCAD cold start is enabled or not.

Reviewer 2: ```suggestion
plugins.anomaly_detection.hcad_cold_start_interpolation.enabled | False | When set to `true`, enables interpolation in high-cardinality anomaly detection (HCAD) cold start.
```

Author: sounds good
